project/src/main/java/ui/screens/BucketListScreen.java

```diff
@@ -0,0 +1,73 @@
+package ui.screens;
+
+import io.appium.java_client.MobileBy;
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.iOSFindBy;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.openqa.selenium.By;
+import ui.BaseUiModule;
+
+import java.util.List;
+
+public class BucketListScreen extends BaseUiModule {
+
+
+    @AndroidFindBy(id = "bucket_cell_root")
+ 	@iOSFindBy(accessibility = "feedCell")
+ 	public List<MobileElement> bucketListContainers;
+
+    @iOSFindBy(uiAutomator = ".collectionViews()[0].cells()[0].textFields()[0]")
```

Comment:
does it work now?
plz check if fields with uiAutomator findBy strategy can be found in XCUITest session